recipes/riboseed/meta.yaml

```diff
@@ -1,47 +1,47 @@
 {% set name = "riboSeed" %}
-{% set version = "0.4.16" %}
+{% set version = "0.4.35" %}
 {% set hash_type = "sha256" %}
-{% set hash_value = "abb680a514058c2233f34a3c574727f61ed65e06b2123964b359683ccf9eb9f5" %}
+{% set hash_value = "9e0233f51babb3c181f1de4afaef34f11dc1d319f2177516f1194654998caa5b" %}
 package:
   name: '{{ name|lower }}'
   version: '{{ version }}'
 
 source:
   fn: '{{ name }}-{{ version }}.tar.gz'
-  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  url: https://test-files.pythonhosted.org/packages/21/59/ef9dbb631f132c312bdd1cc145395050fdd0bfbe46765280c9a157367dcd/riboSeed-0.4.16.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 0
+  entry_points:
+    - ribo = riboSeed.ribo:main
+  number: 1
```

Comment:
the number should start with 0 as this is a new package

Reply:
I have version 0 on my personal channel; is it ok to have identical versions on multiple channels?

Reply:
I believe build numbers on different channels are not relevant